doc/api/console.md

```diff
@@ -139,7 +142,7 @@ the default behavior of `console` in Node.js.
 // new impl for assert without monkey-patching.
 const myConsole = Object.create(console, {
   assert: {
-    value: function assert(assertion, message, ...args) {
+    value(assertion, message, ...args) {
```

Comment:
This will result in `myConsole.assert.name === 'value'`, which may not be desirable. I'd add an exclusion for `func-name-matching`.